Documentation/dev_workflow.md

```diff
@@ -16,143 +16,86 @@ Assumptions:
 
 # Workflow
 
-## Cloning and starting Docker container
-
-If you're testing or working on specific changes, use appropriate branches
-instead of main development ones.
-
-```shell
-git clone git@github.com:TrenchBoot/grub.git
-cd grub
-git checkout intel-txt-mb2
-cd ..
-git clone git@github.com:3mdeb/xen.git
-cd xen
-git checkout aem/develop
-cd ..
-docker run --rm -it -v $PWD/grub:/home/trenchboot/grub \
--v $PWD/xen:/home/trenchboot/xen -w /home/trenchboot \
-ghcr.io/trenchboot/trenchboot-sdk:master /bin/bash
+What follows describes the use of `aem.sh` script from [../scripts/aem/]
+directory. It's assumed to be in current directory, but putting it in `$PATH`
+will work the same. The script already has a `README`, look there for more
+details. The description here will be a bit more streamlined.
+
+## Configuration
+
+Start by creating a directory where you'll be working and copy `aem.sh` there.
+Create configuration file called `aem-cfg.sh`. It's a bash script which `aem.sh`
+sources on startup. You want to adjust the following:
+ * branches and maybe URLs to projects if not working on default ones
+ * IP address of your machine (**don't miss this**)
+ * port to use when starting a local HTTP server in case `8080` is in use
+ * disk and partition devices on DUT (**checking this twice won't hurt**, don't
+   put in device of your USB stick instead of the target disk!)
+ * list of files to send to DUT in case you want to use different names or
+   location on DUT (don't forget to adjust `grub.cfg` accordingly)
+
+```bash
+grub_repo=https://github.com/TrenchBoot/grub.git
+xen_repo=https://github.com/3mdeb/xen.git
```

Comment:
I've moved Xen to TrenchBoot (https://github.com/TrenchBoot/xen/tree/aem), please update URL and branch.